Author: Jasper Grimm
Add `scripts` parameter to setup.py to ensure these are installed to the correct directory
diff -Nru syri-1.4.orig/setup.py syri-1.4/setup.py
--- syri-1.4.orig/setup.py	2021-08-23 11:37:57.000000000 +0100
+++ syri-1.4/setup.py	2021-08-23 11:39:36.000000000 +0100
@@ -19,4 +19,5 @@
         Extension('syri.findsv',['syri/pyxFiles/findsv.pyx']),
         Extension('syri.writeout',['syri/pyxFiles/writeout.pyx'])]),
         packages=["syri","syri.bin", "syri.bin.func"],
+        scripts=['syri/bin/%s' % x for x in ['chroder', 'chrrev.py', 'plotsr', 'regAnno', 'syri']],
         include_dirs=[numpy.get_include()])
